### PR TITLE
Rename healthcare employee to facility role in CSV Uploader

### PR DESCRIPTION
**Pull request by nickrobison-usds**
Remove repurposed field to now convey a person's role in a given facility.
This only changes the CSV uploader, which was lagging behind the GraphQL schema.

### CODE REVIEW

File: backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public void processPersonCSV(InputStream csvStream) throws IOException {
             final Map<String, String> row = valueIterator.next();
 
             final LocalDate patientDOB = LocalDate.parse(row.get("DOB"));
-            List<String> race = new ArrayList<String>();
+            List<String> race = new ArrayList<>();
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java:49]: 👍

**Review by benwarfield-usds — APPROVED** [review]
:shipit:

File: backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java [hunk 2]
```diff
@@ -45,8 +45,9 @@
 	private String email;
 	@Column(nullable = false)
 	private boolean employedInHealthcare;
-	@Column
-	private String role;
+	@Column(nullable = false)
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java:48]: It's nullable in the DB, though—do we want to change that too?

File: backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java [hunk 3]
```diff
@@ -1,22 +1,22 @@
 package gov.cdc.usds.simplereport.db.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonRole;
```
Comment (benwarfield-usds) [inline backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java:6]: I love that IntelliJ and Eclipse have different default orders for organizing exports. Get it together, people! 🤣

**Review by benwarfield-usds — CHANGES_REQUESTED** [review]
Nice! Only issue: it looks like we are asserting that "role" is non-null for the service and persistence layers, but not in the API schema or the DB schema. We should make it consistent throughout, I think.

**Comment by nickrobison-usds** [comment]
Good catch, fixed!

File: backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java [hunk 4]
```diff
@@ -5,12 +5,11 @@
 import java.time.LocalDate;
 import java.util.List;
 
-import gov.cdc.usds.simplereport.db.model.auxiliary.PersonRole;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 
 import gov.cdc.usds.simplereport.db.model.Person;
-import gov.cdc.usds.simplereport.db.repository.BaseRepositoryTest;
+import gov.cdc.usds.simplereport.db.repository	.BaseRepositoryTest;
```
Comment (benwarfield-usds) [inline backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java:12]: This seems unintentional, albeit harmless. 😁

**Review by benwarfield-usds — APPROVED** [review]
:shipit: 

We still have a mild inconsistency between DB and code, but the worst case is that the code gets a little confused at some point, and there are lots of other ways to do that. 😁

**Pull request merged**